### PR TITLE
Code Snippets play

### PR DESCRIPTION
**Pull request by senali-d**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (#803)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

# Screenshots or example output

### CODE REVIEW

File: src/plays/code-snippets/CodeSnippets.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        {({ onStartCapture }) => (
+          <div className="flex flex-col md:flex-row">
+            <div className="w-full md:w-1/2 p-5">
+              <MEditor data={snippets} onChange={handleEditorChange} />
+            </div>
+            <div className="flex flex-col w-full md:w-1/2 p-5">
+              <div className="flex mb-3">
+                <button
+                  className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded mr-2"
+                  onClick={onStartCapture}
```
Comment (atapas) [inline src/plays/code-snippets/CodeSnippets.jsx:41]: Where is the `onStartCapture` function defined?

Comment (senali-d) [reply src/plays/code-snippets/CodeSnippets.jsx:41]: It is in line no 32.
```
 <ScreenCapture onEndCapture={handleScreenCapture}>
        {({ onStartCapture }) => (
          <div className="flex flex-col md:flex-row">
            <div className="w-full md:w-1/2 p-5">
```

**Comment by senali-d** [comment]
Yeah sure. I'll take care of it.

**Comment by atapas** [comment]
> Yeah sure. I'll take care of it.

We still get this error

image

**Pull request closed without merging**